src/blog/2022/07/flowforge-07-released.md

```diff
@@ -0,0 +1,66 @@
+
+[FlowForge Theme](https://github.com/flowforge/flowforge-nr-theme/)
+Now that we have a stronger visual identity in the Forge application we have continued that work throiugh to the Node-RED editor. If you are running a Node-RED 3.0 stack you will see a different theme in the editor. It's still very much Node-RED but just has some subtle hints to tie it back to the FlowForge application. We will continue to iterate on this to further integrate the expereince between FlowForge and Node-RED in both directions.
+![](../images/ff-07-theme.png)
+
+[ProjectTypes](https://github.com/flowforge/flowforge/issues/380)
+The introduction of ProjectTypes is largely an internal feature to support our FlowForge Cloud proposition by being able to have different price points for projects based on the capacity and features availble, While Stacks were a part of this ProjectTypes allows for an aggregation of stacks so that users can change versions of Node-RED while remainging on the same pricing tier.
+
+[Stack Versions](https://github.com/flowforge/flowforge/issues/694)
+This allows an admin to link different stacks together in their lineage so for example when an updated version of Node-RED on a particular stack is deployed any users with projects on that old version will be prompted that their is an update availble, making it even easier to stay up to date with Node-RED versions when you build your flows on FlowForge.
+
+
+## Improvements
+We've made a number of improvements to the overall experience of running FlowForge.
+
+The Team Switch menu has been moved to a more prominent position in the interface, this also makes it easier to see how to create a new team.[https://github.com/flowforge/flowforge/issues/616]
+If you are running your own copy of FlowForge you can now see the version details in the admin pages [https://github.com/flowforge/flowforge/issues/655]
+Device polling is no longer an INFO level message  filling the log on your devices [https://github.com/flowforge/flowforge-device-agent/issues/10]
+
+
+## Bug Fixes
+[Devices now listen on all Interfaces allowing you to run local http servers](https://github.com/flowforge/flowforge-device-agent/issues/7)
+[Fixed an error where a device gets an error  unknown device](https://github.com/flowforge/flowforge-device-agent/issues/7)
+[The Audit Log in the Forge app displays the correct IP when a user logs in to Node-RED](https://github.com/flowforge/flowforge/issues/507)
+[Fixed an error with devices downloading snaphots from legacy stacks](https://github.com/flowforge/flowforge/issues/507)
+[Fixed an error where objects in the node-red log would hang the log page](https://github.com/flowforge/flowforge/issues/735)
+[Fixed an error where the Next Billing Date was incorrect](https://github.com/flowforge/flowforge/issues/745)
+### Upgrading FlowForge
+
+[FlowForge Cloud](https://app.flowforge.com) is already running 0.7 - 
```

Comment:
Should the double new line be removed so it's one paragraph? Ending a sentence/paragraph with `-` seems odd.